.storybook/main.js

```diff
@@ -1,4 +1,13 @@
+const IS_PRODUCTION = process.env.NODE_ENV === 'production'
+
+const stories = ["./**/*.stories.jsx"]
+
+// stories from src are for development purposes only
+if(!IS_PRODUCTION) {
+    stories.push("../src/**/*.stories.tsx")
+}
+
 module.exports = {
-    stories: ["./**/*.stories.jsx", "../src/**/*.stories.tsx"],
-    addons: ["@storybook/preset-typescript"],
```

Comment:
This preset is not necessary anymore and was causing issues that made Storybook break!